src/helm/benchmark/run_specs.py

```diff
@@ -1742,7 +1742,7 @@ def get_lextreme_spec(subset: str) -> RunSpec:
         name=f"lextreme:subset={subset}",
         scenario_spec=scenario_spec,
         adapter_spec=adapter_spec,
-        metric_specs=get_f1_metric_specs(),
+        metric_specs=get_f1_metric_specs() + get_classification_metric_specs(),
```

Comment:
Ideally you would append `classification_metric_specs` only if the subset is classification.